src/components/Stats/Stats.tsx

```diff
@@ -0,0 +1,53 @@
+/** @jsx h */
+
+import { h } from 'preact';
+import cx from 'classnames';
+import { StatsCSSClasses, StatsTemplates } from '../../widgets/stats/stats';
+import Template from '../Template/Template';
+
+type StatsProps = {
+  cssClasses: StatsCSSClasses;
+  templateProps: {
+    [key: string]: any;
+    templates: StatsTemplates;
+  };
+  hitsPerPage: number | undefined;
+  nbHits: number;
+  nbSortedHits: number | undefined;
+  areHitsSorted: boolean;
+  nbPages: number;
+  page: number;
+  processingTimeMS: number;
+  query: string;
+};
+
+const Stats = ({
+  nbHits,
+  nbSortedHits,
+  cssClasses,
+  templateProps,
+  ...rest
```

Comment:
I've used `...rest` to avoid re-writing all the props, should it stay as before?